unit-tests/src/test/scala/scala/scalanative/posix/TimeTest.scala

```diff
@@ -20,14 +21,26 @@ class TimeTest {
   val now_time_t: time_t = scala.scalanative.posix.time.time(null)
   val epoch: time_t      = 0L
 
+  // some of the tests (the ones that call localtime) need
+  // for the daylight saving time to not be in effect
+  val timeIsDst: Boolean = {
```

Comment:
I suggest flipping the sense of `timeIsDst()` to `timeIsStandard()` and 
making the corresponding change in the localtime* tests.
See the coment at line 32 below.